tests/queries/0_stateless/03403_session_same_name_race.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+export SESSION="03403_session_${CLICKHOUSE_DATABASE}"
+export SESSION_ID="${SESSION}_$RANDOM.$RANDOM"
+export SETTINGS="session_id=$SESSION_ID&session_timeout=2&throw_on_unsupported_query_inside_transaction=0"
```

Comment:
`throw_on_unsupported_query_inside_transaction=0` - what is it for? It looks like it doesn't belong to this test.

Also please add a comment to this test with some description for the scenario you test here.

Reply:
Some leftover from `02435_rollback_cancelled_queries`, thanks

Reply:
Done